lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,25 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  The `Macro.escape/2` function is sometimes confused with `Kernel.quote/2`,
+  because the above examples behave the same with both. The key difference is
+  best illustrated when the value to escape is stored in a variable.
+
+  ## Comparison to `Kernel.quote/2`
+
+      iex> quote do: {:a, :b, :c}
```

Comment:
Please use what the formatter would output which if I am not wrong should be

```elixir
iex> quote(do: {:a, :b, :c})
```

I don't have a computer with Elixir around, but you can verify that by doing `"quote do: {:a, :b, :c}" |> Code.format_string!() |> IO.puts()` in an IEx session. Do this for all `quote`s please :)

Reply:
I ran all the code in the examples through the formatter and got this:

```elixir
quote do: {:a, :b, :c}
value = {:a, :b, :c}
Macro.escape(value)
quote do: value
value = {:a, :b, :c}
quote do: unquote(value)
```

So the only change appears to be the variable bindings, however I'm not sure if I can split variable binding in doctests. Can you confirm?

Reply:
Even so, I'm happy to wrap the quote calls in parens! I don't have a strong opinion :)

Reply:
I'm mistaken. The separate variable binding works. Not sure what I did wrong. Will update!